src/games/strategy/triplea/ResourceLoader.java

```diff
@@ -153,23 +152,33 @@ private ResourceLoader(final String[] paths) {
         throw new IllegalStateException(e.getMessage());
       }
     }
+    resourceLocationTracker = new ResourceLocationTracker(mapName, urls);
     // Note: URLClassLoader does not always respect the ordering of the search URLs
     // To solve this we will get all matching paths and then filter by what matched
     // the assets folder.
     m_loader = new URLClassLoader(urls);
   }
 
+  public void close() {
```

Comment:
Was really hard to find the class constructor, notice the constructor was moved above this method.